io.codeberg.redwn.sync_o_theft.yml

```diff
@@ -0,0 +1,29 @@
+id: io.codeberg.redwn.sync_o_theft
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: 'run_sync_o_theft.sh'
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+modules:
+   - name: sync_o_theft
+     buildsystem: simple
+     build-commands:
+       - install -D run_sync_o_theft.sh /app/bin/run_sync_o_theft.sh
+       - install -D Sync-O-Theft.x86_64 /app/share/sync_o_theft/Sync-O-Theft.x86_64
```

Comment:
so what happen when building the aarch64 package?

Reply:
https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source

Reply:
> Building with Unity will require some license activation, as far as I know there is no work around that for now.

why? how are you building it?